scripts/util.sh

```diff
@@ -37,7 +37,7 @@ function util::wait-for-condition() {
   local error_msg="[ERROR] Timeout waiting for ${msg}"
 
   local counter=0
-  while ! ${condition}; do
+  while ! eval ${condition}; do
```

Comment:
just curious, how did you find that this needed to be changed?

Reply:
I discovered this when the `condition` argument being passed to this function included a redirection such as:
``kubectl --context cluster${i} get --raw=/healthz &> /dev/null``

What happened is the redirection argument `&> /dev/null` was being passed to `kubectl` and it didn't know what to do with it. Using `eval` allows the `condition` args to be properly constructed into a single command and passed to the shell resulting in the proper execution.